hack/test-e2e.sh

```diff
@@ -5,7 +5,9 @@ if [ -n "${IMAGE_CLUSTER_LOGGING_OPERATOR:-}" ] ; then
   source "$(dirname $0)/common"
 fi
 
+IMAGE_STREAM_CLUSTER_LOGGING_OPERATOR=$(oc get is origin-cluster-logging-operator -n openshift-logging -o jsonpath='{.status.dockerImageRepository}')
```

Comment:
This doesn't work for me in a CI cluster e.g.
```
oc get project
NAME                 DISPLAY NAME                                                                         STATUS
ci-op-m57t3frq       ci-op-m57t3frq - pull-ci-openshift-origin-aggregated-logging-master-full-integ-aws   Active
```
the namespace name is unique for each ci run and is available in the env. var. `$OPENSHIFT_BUILD_NAMESPACE`.
```
oc -n ci-op-m57t3frq get is
NAME       DOCKER REPO                                             TAGS                                                                   UPDATED
pipeline   registry.svc.ci.openshift.org/ci-op-m57t3frq/pipeline   logging-kibana5,logging-fluentd,logging-elasticsearch5 + 6 more...     About an hour ago
release    registry.svc.ci.openshift.org/ci-op-m57t3frq/release    latest                                                                 About an hour ago
stable     registry.svc.ci.openshift.org/ci-op-m57t3frq/stable     logging-kibana5,logging-fluentd,logging-elasticsearch5 + 108 more...   About an hour ago
```
I think ci always uses `pipeline` as the name of the `is`.
```
reg=$( oc -n ci-op-m57t3frq get is pipeline -o jsonpath='{.status.dockerImageRepository}' )
echo $reg
docker-registry.default.svc:5000/ci-op-m57t3frq/pipeline
```
then I think the `image:` will be something like `docker-registry.default.svc:5000/ci-op-m57t3frq/pipeline:cluster-logging-operator` - note - without the `origin-`